### PR TITLE
[Docs] update Japanese translation of feature_advanced_keycodes.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

update Japanese translation of docs/feature_advanced_keycodes.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

**Review by shelaf — APPROVED** [review]

File: docs/ja/feature_advanced_keycodes.md [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
+| `LCTL(kc)` | `C(kc)`                            | 左 Control を押しながら `kc` を押します。                           |
+| `LSFT(kc)` | `S(kc)`                            | 左 Shift を押しながら `kc` を押します。                             |
+| `LALT(kc)` | `A(kc)`, `LOPT(kc)`                | 左 Alt を押しながら `kc`を押します。                                |
+| `LGUI(kc)` | `G(kc)`, `LCMD(kc)`, `LWIN(kc)`    | 左 GUI を押しながら `kc` を押します。                               |
+| `RCTL(kc)` |                                    | 右 Control を押しながら `kc` を押します。                           |
+| `RSFT(kc)` |                                    | 右 Shift を押しながら `kc` を押します。                             |
+| `RALT(kc)` | `ROPT(kc)`, `ALGR(kc)`             | 右 Alt を押しながら `kc` を押します。                               |
+| `RGUI(kc)` | `RCMD(kc)`, `LWIN(kc)`             | 右 GUI を押しながら `kc` を押します。                               |
+| `LSG(kc)`  | `SGUI(kc)`, `SCMD(kc)`, `SWIN(kc)` | 左 Shift と左 GUI を押しながら `kc` を押します。                    |
+| `LAG(kc)`  |                                    | 左 Left と左 GUI を押しながら `kc` を押します。                     |
```
Comment (s-show) [inline docs/ja/feature_advanced_keycodes.md:21]:
```suggestion
| `LAG(kc)`  |                                    | 左 Alt と左 GUI を押しながら `kc` を押します。                     |
```

Comment (umi-umi) [reply docs/ja/feature_advanced_keycodes.md]: ご指摘ありがとうございます！

**Review by s-show — APPROVED** [review]

**Review by mtei — APPROVED** [review]

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks!

**Pull request merged**